packages/weex-vue-framework/factory.js

```diff
@@ -175,23 +175,6 @@ var hyphenate = cached(function (str) {
   return str.replace(hyphenateRE, '-$1').toLowerCase()
 });
 
-/**
- * Simple bind, faster than native
- */
-function bind (fn, ctx) {
-  function boundFn (a) {
-    var l = arguments.length;
-    return l
-      ? l > 1
-        ? fn.apply(ctx, arguments)
-        : fn.call(ctx, a)
-      : fn.call(ctx)
-  }
-  // record original fn length
-  boundFn._length = fn.length;
-  return boundFn
-}
-
 /**
```

Comment:
This file is generated by `npm run build:weex`, no need to patch this.

Reply:
Other than code patch, will using native bind benefit weex? I wonder.

Reply:
In my opinion, using js tricks to optimise native APIs is not a good solution. JS engines are changing all the time, this trick may not always efficient.

For Weex, it's possible to optimise the JS engine directly if the native bind has performance issues.